susi_python/main.py

```diff
@@ -5,11 +5,13 @@
 import os
 from .response_parser import *
 from uuid import getnode as get_mac
+import logging
 
+logger = logging.getLogger(__name__)
 
-class Main:
+class SusiMain:
```

Comment:
Sorry to be so picky, but again I think `SusiMain` is not a good naming. This module does nothing **main**, just provides access to the SUSI Server. I think the proper name would be `SusiServer`!

Why, look at the code example. Currenty we would have
```
import susi_python
susi_main.ask('...')
```
which sounds strange, because the **main** part to me is `susi_linux`. If we use `SusiServer` and `susi_server` instead, the code would look like:
```
import susi_python
susi_server.ask('....')
```
which is somehow *what we are doing* ...

WDYT?

Reply:
@norbusan , susi_server is refered to the Java Server here. So, might cause a lot of confusions here.
Should I name it just `SUSI`?

Reply:
But it is about connecting to the susi server, right. It is so to say a local tunnel to the server.

But ok, SUSI/susi alone is fine with me!

Reply:
I think we can name it `SUSIClient`

Reply:
Yes, `SUSIClient` sounds good to me.

Reply:
Ok, fine with me, whatever. Please go ahead and commit the changes.

Reply:
@norbusan @hongquan , updated